function XRefEntryException(t){_classCallCheck(this,XRefEntryException);return r.call(this,t,"XRefEntryException")}